docker/dbt/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fishtownanalytics/dbt:0.21.0
+FROM xemuliam/dbt:1.0.0
```

Comment:
I would advise against doing this.
Let's wait until the official docker image for `dbt` lands.

Reply:
yes I agree, it's used just to not stop the work on the upgrading to v1.0.0 

as an alternative, we can also build our own image

Reply:
Official dbt-core docker image is already avaliable. 
I reverted a previous commit.